### PR TITLE
[rubocop] Style fixes for 0.58.2

### PR DESCRIPTION
**Pull request by harman28**
Fixes style issues that were causing [build to fail](https://travis-ci.org/razorpay/razorpay-ruby/jobs/432676004).

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -19,3 +19,5 @@ Naming/UncommunicativeMethodParamName:
   MinNameLength: 1
   AllowNamesEndingInNumbers: false
 
+Naming/FileName:
+  Exclude: ['razorpay-ruby.gemspec']
```
Comment (tessie) [inline .rubocop.yml]: Why is this needed?

Comment (harman28) [reply .rubocop.yml]: Fails with
```
razorpay-ruby.gemspec:1:1: C: Naming/FileName: The name of this source file (razorpay-ruby.gemspec) should use snake_case.
lib = File.expand_path('lib', Dir.pwd)
```
